azure-pipelines.yml

```diff
@@ -554,6 +554,6 @@ jobs:
             --build_tests_only=false \
             --define DISABLE_VERILATOR_BUILD=true \
             --test_tag_filters=-broken,-cw310,-verilator,-dv \
-            --build_tag_filters=-verilator,-test_suite \
+            --build_tag_filters=-verilator,-cw310,-dv \
```

Comment:
That wasn't my intent here, but I'll be back to clean it up so if this doesn't break anything I'm fine with it.
I was hoping to exclude the test_suite as well as the verilator tests, but let the wildcard matching build the dv and cw310 tests so we'd know if we couldn't build OT binaries. This will cause us not to check the binaries AFAIK.

Reply:
I'd prefer the -test_suite just be dropped. @mcy has a solution to verilator that never really became redundant.